src/index.html

```diff
@@ -4,6 +4,21 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- OpenGraph -->
+    <meta property="og:title" content="ESA - Climate from space" />
+    <meta property="og:type" content="website" />
+    <meta
+      property="og:image"
+      content="https://storage.googleapis.com/esa-cfs-storage/0.6.1/stories/a-passage-opens/assets/sst_large_18.jpg"
```

Comment:
We need to make these urls dynamic. As this has to be in place before any javascript kicks in we have to insert the correct url at build time.
One solution would be to set the environment variable NODE_ENV to 'live' and then insert the "live" url. We can discuss this in person if you like...